### PR TITLE
Make brave unusable to encourage updating

### PR DESCRIPTION
**Pull request by diracdeltas**
This PR replaces the newtab page with the notice to update Brave and
makes the urlbar unusable.

Test plan:
1. Launch Brave without brave-core installed
2. You should see a notice to update brave.
3. Clicking on it should download Brave.
4. The urlbar should not be usable.
5. With brave-core now installed, launch the old Brave again.
6. Now you should see a notice to launch the newer Brave installation.
7. The urlbar should still be unusable.

Test plan for disabling tab restoration:
1. Open previous version of muon brave and open some tabs/windows
2. Update to this version
3. You should not see any of your tabs/windows restored

Screenshot:
Screen Shot 2019-03-28 at 4 23 38 PM

## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request [as needed](https://github.com/brave/browser-laptop/wiki/Pull-request-process).
- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review). (Ask a Brave employee to help if you cannot access this document.)

## Reviewer Checklist:

- [ ] Request a security/privacy review [as needed](https://github.com/brave/handbook/blob/master/development/security.md#how-to-request-a-security-review) if one was not already requested.

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

**Comment by diracdeltas** [comment]
just realized, i should disable tab restoration too

**Comment by diracdeltas** [comment]
tab restoration disabling is done now

**Comment by bsclifton** [comment]
@diracdeltas you can still open private tabs, which don't show our new messaging

Should we disable these too?

**Comment by diracdeltas** [comment]
> you can still open private/Tor tabs, which don't show our new messaging

good catch, will disable private tabs

**Comment by diracdeltas** [comment]
@bsclifton changed it so that private/tor tabs now also show the notice

File: app/renderer/components/preferences/syncTab.js [hunk 1]
```diff
@@ -500,9 +500,9 @@ class SyncTab extends ImmutableComponent {
             settingsListTitle: true
           })}>
             Please download the <a className={css(commonStyles.linkText)}
-            onClick={aboutActions.createTabRequested.bind(null, {
-              url: 'https://brave.com/download'
-            })}>new version of Brave for desktop</a>.
+              onClick={aboutActions.createTabRequested.bind(null, {
+                url: 'https://brave.com/download'
+              })}>new version of Brave for desktop</a>.
```
Comment (diracdeltas) [inline app/renderer/components/preferences/syncTab.js]: this was to fix some linter errors from a previous commit

**Pull request merged**